cmake/TaichiCore.cmake

```diff
@@ -200,6 +200,10 @@ if(DEFINED ENV{LLVM_DIR})
     message("Getting LLVM_DIR=${LLVM_DIR} from the environment variable")
 endif()
 
+if(WIN)
+list(APPEND CMAKE_PREFIX_PATH "C:/Program Files(x86)/LLVM")
```

Comment:
I don't think we should make such assumption about where the LLVM lib is installed